x-pack/legacy/plugins/alerting_ui/np_ready/public/plugin.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { unmountComponentAtNode } from 'react-dom';
+import { PluginInitializerContext, CoreSetup, CoreStart, Plugin } from 'src/core/public';
+import { i18n } from '@kbn/i18n';
+import routes from 'ui/routes';
+
+import template from '../../public/index.html';
+import { renderReact } from './application';
+import { BASE_PATH } from './application/constants';
+import { breadcrumbService } from './application/lib/breadcrumb';
+import { docTitleService } from './application/lib/doc_title';
+
+export type Setup = void;
+export type Start = void;
+
+const REACT_ROOT_ID = 'alertingRoot';
+
+export class ActionsPlugin implements Plugin<Setup, Start> {
```

Comment:
Maybe it makes sense to name the plugin AlertingUiPlugin, because ActionsPlugin was come up when we considered to have a separate Alerts and Actions plugins

Reply:
Fixed in https://github.com/elastic/kibana/pull/47843/commits/dfca1a1fa35cc4567bc0efd03e14ce9ffc38081a. Note I renamed it to `Plugin` instead to match filename.